e2e/README.md

```diff
@@ -140,12 +152,12 @@ cluster or you can pass `kubeconfig`flag while running tests.
 
 Functional tests are run by the `go test` command.
 
- ```console
- $go test ./e2e/ -timeout=20m -v -mod=vendor
- ```
+```console
+$go test ./e2e/ -timeout=20m -v -mod=vendor
```

Comment:
This is wrong (and was a;ready before you modified it), `$go` would be a variable, it definitely needs a space between `$` and `go` here. Check the other parts of the doc where `[$] command` is used, maybe apply the same in the new additions/modifications?

Reply:
ok! alright, let me try `[$]`.

Reply:
Neils, I'm convinced with https://updownpress.github.io/markdown-lint#MD014

The gist is, `The dollar signs are unnecessary, it is easier to copy and paste and less noisy if the dollar signs are omitted. Especially when the command doesn't list the output.`


@nixpanic are we good at removing `$` from the commands, at the advantage of easy copy-paste?

@Madhu-1 @humblec thoughts?

Reply:
yes, I agree that they are not needed. However we do have them in all (most?) other documents. My main concern is about the style we want to use, and that we should use it everywhere. I do not care much about what style we use in the end (but `$go` is just wrong).

Reply:
If we want to stick to this style (i.e. no `$` at the beginning for commands that don't have output) I can generate a patch and add it to this PR, to keep the style uniform across all the md files.

Reply:
@Madhu-1 @humblec thoughts ?

Reply:
Sounds good. Please update the coding guide also so that we can have a standard

Reply:
:+1: